spec/system/admin/tenants_spec.rb

```diff
@@ -40,6 +40,52 @@
       expect(page).to have_current_path root_path
       expect(page).to have_link "Sign in"
     end
+
+    scenario "Copies the current user as the new tenant administrator" do
+      user = create(:user, password: "secret_password", password_confirmation: "secret_password")
```

Comment:
The `password_confirmation` part doesn't seem necessary :thinking:.

```suggestion
      user = create(:user, password: "secret_password")
```